shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java

```diff
@@ -59,6 +61,14 @@ public final void init() throws Exception {
         try (Connection connection = getTargetDataSource().getConnection()) {
             executeInitSQLs(connection);
         }
+        intervalTime();
+    }
+    
+    @SneakyThrows(InterruptedException.class)
+    private void intervalTime() {
```

Comment:
the method name is not clear, method name should be Verb + Nouns

Reply:
I will fix it.